### PR TITLE
PR 13964 5 6

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
IDs for persons related to UM 042

### CODE REVIEW

File: new/PRS13965MehdentaMaryam.xml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1">ምኅፅንተ፡ ማርያም፡</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1">Mǝḫḍǝnta Māryām</persName>
+                    <faith type="EOTC"/>
+                    <occupation type="art"/>
```
Comment (eu-genia) [inline new/PRS13965MehdentaMaryam.xml:51]: I do not think we assign "art" to scribes

File: new/PRS13966WalattaKrestos.xml [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+            </t:langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="DN" when="2023-01-19">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
```
Comment (eu-genia) [inline new/PRS13966WalattaKrestos.xml:47]: I assume she is a lady, so sex="2"

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: new/PRS13964KeflaMaryam.xml [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1">ክፍለ፡ ማርያም፡</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1">Kǝfla Māryām</persName>
+                    <faith type="EOTC"/>
+                    <occupation type="ecclesiastic"/>
+                    <birth/>
+                    <t:death xmlns:t="http://www.tei-c.org/ns/1.0"/>
```
Comment (thea-m) [inline new/PRS13964KeflaMaryam.xml:53]: Can you delete the empty birth and death elements?

File: new/PRS13964KeflaMaryam.xml [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
+            </abstract>
+            <t:langUsage xmlns:t="http://www.tei-c.org/ns/1.0">
```
Comment (thea-m) [inline new/PRS13964KeflaMaryam.xml:35]: can you delete t: (which was inserted during the creation via the app)?

**Review by thea-m — CHANGES_REQUESTED** [review]

File: new/PRS13964KeflaMaryam.xml (hunk 3 above)
Comment (DenisNosnitsin1970) [reply new/PRS13964KeflaMaryam.xml:53]: Ok

File: new/PRS13964KeflaMaryam.xml (hunk 4 above)
Comment (DenisNosnitsin1970) [reply new/PRS13964KeflaMaryam.xml:35]: Ok, I do.

Comment (eu-genia) [reply new/PRS13964KeflaMaryam.xml:35]: and also xmlns etc. it must be simply <langUsage>

File: new/PRS13964KeflaMaryam.xml (hunk 3 above)
Comment (eu-genia) [reply new/PRS13964KeflaMaryam.xml]: and the link, it must be simply `<death>`

File: new/PRS13965MehdentaMaryam.xml [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
+            </abstract>
+            <t:langUsage xmlns:t="http://www.tei-c.org/ns/1.0">
```
Comment (eu-genia) [inline new/PRS13965MehdentaMaryam.xml:35]: `<langUsage>`

File: new/PRS13966WalattaKrestos.xml [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
+            </abstract>
+            <t:langUsage xmlns:t="http://www.tei-c.org/ns/1.0">
```
Comment (eu-genia) [inline new/PRS13966WalattaKrestos.xml:35]: `<langUsage>`

File: new/PRS13966WalattaKrestos.xml [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1">ወለተ፡ ክርስቶስ፡ </persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1">Walatta Krǝstos</persName>
+                    <faith type="EOTC"/>
+                    <occupation type="other"/>
+                    <birth/>
+                    <t:death xmlns:t="http://www.tei-c.org/ns/1.0"/>
```
Comment (eu-genia) [inline new/PRS13966WalattaKrestos.xml]: `<death>`

**Review by eu-genia — CHANGES_REQUESTED** [review]

**Review by eu-genia — APPROVED** [review]

**Pull request merged**